CONTRIBUTING.md

```diff
@@ -104,9 +104,9 @@ This repository is used by Chart developers for maintaining the official charts
 * Provide a secure default configuration
 * Do not leverage alpha features of Kubernetes
 * Includes a [NOTES.txt](https://github.com/helm/helm/blob/master/docs/charts.md#chart-license-readme-and-notes) explaining how to use the application after install
-* Follows [best practices](https://github.com/helm/helm/tree/master/docs/chart_best_practices)
-  (especially for [labels](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md)
-  and [values](https://github.com/helm/helm/blob/master/docs/chart_best_practices/values.md))
+* Follows [best practices](https://github.com/helm/helm-www/tree/master/content/en/docs/chart_best_practices)
+  (especially for [labels](https://github.com/helm/helm-www/blob/master/content/en/docs/chart_best_practices/labels.md)
+  and [values](https://github.com/helm/helm-www/blob/master/content/en/docs/chart_best_practices/values.md))
```

Comment:
I think it would be better to link to the rendered docs site at https://helm.sh/docs instead.

Reply:
Doh, this makes sense